src/record/delete/usecases/delete/validator.ts

```diff
@@ -0,0 +1,41 @@
+import type { KintoneRestAPIClient } from "@kintone/rest-api-client";
+import type { RecordNumber } from "../../types/field";
+import type { InvalidRecordNumber } from "../../types/record";
```

Comment:
IMO: You can move `InvalidRecordNumber` to this file, because it is a intermediate representation to create error.

Reply:
I removed this type from PR #220